src/main/java/frc/trigon/robot/poseestimation/relativerobotposesource/io/RelativeRobotPoseSourceT265IO.java

```diff
@@ -0,0 +1,46 @@
+package frc.trigon.robot.poseestimation.relativerobotposesource.io;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.networktables.*;
+import frc.trigon.robot.poseestimation.relativerobotposesource.RelativeRobotPoseSourceIO;
+import frc.trigon.robot.poseestimation.relativerobotposesource.RelativeRobotPoseSourceInputsAutoLogged;
+
+public class RelativeRobotPoseSourceT265IO extends RelativeRobotPoseSourceIO {
+    private final IntegerSubscriber framesPerSecond;
+    private final DoubleSubscriber batteryPercentage;
+    private final FloatArraySubscriber positionMeters;
+    private final FloatArraySubscriber rotationRadians;
+
+    public RelativeRobotPoseSourceT265IO(String hostname) {
+        final NetworkTable t265NetworkTable = NetworkTableInstance.getDefault().getTable(hostname);
+
+        framesPerSecond = t265NetworkTable.getIntegerTopic("FPS").subscribe(0);
+        batteryPercentage = t265NetworkTable.getDoubleTopic("BatteryPercentage").subscribe(0.0f);
+        positionMeters = t265NetworkTable.getFloatArrayTopic("PositionMeters").subscribe(new float[]{0.0f, 0.0f, 0.0f});
+        rotationRadians = t265NetworkTable.getFloatArrayTopic("RotationRadians").subscribe(new float[]{0.0f, 0.0f, 0.0f});
```

Comment:
make all these one json string, as I said before
assume they are sent as one array of x and y, and one value for the rotation